### DISCUSSION TITLE
Only last added cronjob working.

### DISCUSSION THREAD

**Discussion by Gamei666** [ltsp/ltsp#705] [Q&A] [answered]
Hello i have multiple cronjobs configured in ltsp.conf it looks like following:
[clients]
CRONTAB_x="0 12 * * * root /sbin/reboot"
CRONTAB_x="0 0 * * * root /sbin/reboot"
CRONTAB_x="@reboot /1.48/runscript.sh"
Everything worked fine previously but now i see in clients /etc/cron.d/ltsp only last added job. Is syntax changed or something happen? Please help.

**Comment by intelliant01** [comment, marked as answer]
Try replacing x with 01,02,03.

**Reply by Gamei666** [reply]
Yep, that worked.  Thank you.
